src/redturtle/volto/restapi/services/copymove/copymove.py

```diff
@@ -0,0 +1,21 @@
+from plone.restapi.services.copymove.copymove import Copy as BaseCopy
+from plone.restapi.services.copymove.copymove import Move as BaseMove
+from redturtle.volto.adapters.namechooser import check_alias
+
+
+class Copy(BaseCopy):
+    """Copies existing content objects."""
+
+    def clipboard(self, parent, ids):
+        for id in ids:
+            check_alias(context=self.context, id=id)
```

Comment:
mmh sei sicuro che qui ci va self.context e non parent ?

```suggestion
            check_alias(context=parent, id=id)
```

Reply:
si perché il contesto è già il parent del contenuto che si vuole spostare. E' dove viene chiamato l'endpoint.

Reply:
questo tu lo dai per scontato, "parent" invece, dal nome, è esplicitamente il contenitore